examples/redwoodjs/redwood.toml

```diff
@@ -0,0 +1,7 @@
+[web]
+  port = 8910
+  apiProxyPath = "/.netlify/functions"
```

Comment:
Do you know what `apiProxyPath` is?

The docs just say "TODO" 🤔 

![image](https://user-images.githubusercontent.com/229881/84537225-57335600-acbd-11ea-9dbb-873321db1c10.png)

Reply:
No idea…